database/main.py

```diff
@@ -0,0 +1,31 @@
+# main.py
+
+import sqlalchemy as sa
+
+from fastapi import FastAPI
+from sqlalchemy.orm import sessionmaker
+
+from config import DATABASE_URL
+from release_dao import readRelease, getkeys
+
+engine = sa.create_engine(DATABASE_URL)
+Session = sessionmaker(bind=engine)
+
+s = Session()
+
+app = FastAPI()
+
+
+@app.get("/releases")
+async def getall():
+    objectString = "{ "

```

Comment:
There is no need to assemble the JSON payload ourselves. I have found an interesting utility from Fast API:

Just import these modules:
```
from fastapi.encoders import jsonable_encoder
from fastapi.responses import JSONResponse
```

and use them like this:
```
    releases_to_return = []    
    keylist = getkeys()
    for id in keylist:
      r = jsonable_encoder(readRelease(id))
      log.info(f"successfully obtained a release instance from the DAO layer: {r}")
      releases_to_return.append(r)

    return JSONResponse(content={ "releases": releases_to_return })
```